gaphor/ui/mainwindow.py

```diff
@@ -777,7 +662,7 @@ def tree_view_create_package(self):
 
     @action(
         name="tree-view-delete-package",
-        label=_("Delete pac_kage"),
+        label=_("Delete package"),
```

Comment:
Why did you drop the underscore? This used to be used for quick access keys in the menu. Is that no longer working like that?

Reply:
Thanks for the correction, that's definitely why I needed your help reviewing this. I just figured it was a typo 😄